Spark_Application/spark_batch_gold.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col, avg, count, when, max, round, stddev
+from pyspark.sql.types import DateType
+
+os.environ["PYSPARK_PYTHON"] = sys.executable
+os.environ["PYSPARK_DRIVER_PYTHON"] = sys.executable
+
+spark = (
+    SparkSession.builder.appName("League Of Legend Player Batch Silver Layer")
+    .master("yarn")
+    .config("spark.home", "/usr/lib/spark")
+    .enableHiveSupport()
+    .getOrCreate()
+)
+
+
+filter_time = spark.conf.get("spark.config.partition.time")
+
+silver_playerlogs_df = spark.read.parquet(
+    "s3://sjm-simple-data/silver_analysis_riot/"
+).filter("create_room_date='" + filter_time + "'")
```

Comment:
```suggestion
silver_playerlogs_df = spark.sql(
    f"SELECT * FROM mydatabase.silver_analysis_riot WHERE create_room_date='{filter_time}'"
)
```
앞서 bronze, silver 모두 Glue 테이블로 생성했다면 직접 경로를 전달하는 것보다
SQL을 통해 DataFrame을 가져오는게 어떨까요?

Reply:
변경했습니다~